src/status_im/contexts/shell/view.cljs

```diff
@@ -15,6 +15,15 @@
     (rf/dispatch [:navigate-back])
     true))
 
+;; A hidden view-id-tracker view, required for e2e testing
+(defn view-id-tracker
+  []
+  (let [view-id (rf/sub [:view-id])]
+    [rn/text
+     {:accessibility-label :view-id-tracker
+      :style {:position :absolute :width 0 :height 0 :z-index -1 :color :transparent}}
+     view-id]))
```

Comment:
In theory, position and z-index should be sufficient. Should I remove the unnecessary ones? It's just a precaution to avoid issues like text being partially visible while the screen is loading or other edge cases.